cmds/exp/sed/sed.go

```diff
@@ -0,0 +1,130 @@
+// Copyright 2023 the u-root Authors. All rights reserved
+// Use of this source code is governed by a BSD-style
+// license that can be found in the LICENSE file.
+
+// By Ahmed Kamal <email.ahmedkamal@googlemail.com>
+
+package main
+
+import (
+	"bufio"
+	"flag"
+	"fmt"
+	"io"
+	"os"
+	"path"
+	"strings"
+)
+
+type transform struct {
+	from string
+	to   string
+}
+
+type transforms []transform
+
+func (t *transforms) String() string {
+	return fmt.Sprint(*t)
+}
+
+func (t *transforms) Set(value string) error {
```

Comment:
I believe sed command parse should be more complicated:

for example this valid command doesn't work:
`./sed -e 's/m.ngo/apple/' FILE`

Also `./sed 's/mango/apple' FILE` don't work either, but it should according to POSIX spec.

So probably the whole sed command parser is complicated, so it's ok to start small. But it need to be aware of regular expression, so I don't think strings.Split will be enough.